crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -114,19 +119,52 @@ pub struct BinnedRenderPhaseBatch {
 }
 
 /// Information about the unbatchable entities in a bin.
-pub struct UnbatchableBinnedEntities {
+pub(crate) struct UnbatchableBinnedEntities {
     /// The entities.
-    pub entities: Vec<Entity>,
+    pub(crate) entities: Vec<Entity>,
 
-    /// Each entity's dynamic offset.
+    pub(crate) dynamic_offsets: UnbatchableBinnedEntityDynamicOffsets,
```

Comment:
I feel like this member could be named better as it isn’t just/isn’t always dynamic offsets. It’s primarily the instance index, and secondarily the dynamic offset if needed. I don’t remember the type that GpuArrayBuffer push returns, off the top of my head. But that type’s name could give a clue. Unless it too needs renaming. :)